src/privileged/trackers/api.js

```diff
@@ -90,12 +90,17 @@ this.trackers = class extends ExtensionAPI {
         },
         async pageBeforeUnloadCallback(e) {
           const tabId = tabTracker.getBrowserTabId(e.target);
+          let uri;
           try {
+            uri = Services.io.newURI(e.data.telemetryData.completeLocation);
```

Comment:
Being paranoid here ... can we remove `completeLocation` from the event data after we're done with it here? I'm not sure if some machinery might accidentally post this in an error stacktrace in a crash report or something.